asctb-api/src/index.ts

```diff
@@ -13,11 +12,10 @@ import { setupPlaygroundRoutes } from './routes/playground';
 import { setupStaticPageRoutes } from './routes/static-pages';
 import { routeCache } from './utils/route-caching';
 
-
 export const app = express();
 app.use(cors());
-app.use(bodyParser.urlencoded({ extended: false }));
-app.use(bodyParser.json());
+app.use(express.urlencoded({ extended: false }));
```

Comment:
Are you sure we aren't supposed to use bodyParser? I thought they had moved some stuff out of express and you were supposed to use them fro separate modules.

Reply:
Yes, they were removed from express in version 4.0.0 but readded in 4.16.0 https://github.com/expressjs/express/releases/tag/4.16.0

We are using express 4.17.0 and it was giving me a warning as  `deprecated body-parser`
https://stackoverflow.com/questions/47232187/express-json-vs-bodyparser-json

Reply:
Ahh ok :+1: